[反馈]: ws超时

### 部署平台

Docker

### 版本号

0.6.3

### 客户端/工具

docker

### 请求接口

None

### 问题/请求描述

<img width="3263" height="1121" alt="Image" src="https://github.com/user-attachments/assets/9cd35501-113f-4ea6-9918-30b07afe2609" />
大佬，用着用着就用不了了，怎么解决这个问题呢？

### 日志信息（可选）

```text
[WARN] 2026-02-12 18:46:49.722 [Asia/Shanghai] [ProxySystem] - ⚠️ [System] Attempting to switch to alternative account...

[ERROR] 2026-02-12 18:46:49.722 [Asia/Shanghai] [ProxySystem] - ❌ [System] Failed to switch to alternative account: Switch already in progress.

[INFO] 2026-02-12 18:46:49.723 [Asia/Shanghai] [ProxySystem] - [System] Resetting isSystemBusy=false in recovery finally block

[INFO] 2026-02-12 18:46:54.659 [Asia/Shanghai] [ProxySystem] - [Browser] Attempting reconnection 1 attempt...

[INFO] 2026-02-12 18:46:54.660 [Asia/Shanghai] [ProxySystem] - [Browser] Connecting to server: ws://127.0.0.1:9998

[INFO] 2026-02-12 18:47:57.906 [Asia/Shanghai] [ProxySystem] - [Browser]  WebSocket connection error: JSHandle@object

[INFO] 2026-02-12 18:47:57.906 [Asia/Shanghai] [ProxySystem] - [Browser] ❌ Connection disconnected, preparing to reconnect...

[ERROR] 2026-02-12 18:47:57.905 [Asia/Shanghai] [ProxySystem] - [Browser Page Error] [JavaScript Error: "Camoufox can’t establish a connection to the server at ws://127.0.0.1:9998/." {file: "blob:https://4bub4uor6w4pdu09lmbekx7zvne7hx07ezdyv2uci2kvvaixq7-h868144788.scf.usercontent.goog/afaab152-c14f-4acb-bb1d-2961e8bb5264" line: 287}]

[INFO] 2026-02-12 18:48:03.304 [Asia/Shanghai] [ProxySystem] - [Browser] Attempting reconnection 2 attempt...

[INFO] 2026-02-12 18:48:03.305 [Asia/Shanghai] [ProxySystem] - [Browser] Connecting to server: ws://127.0.0.1:9998

[ERROR] 2026-02-12 18:49:03.876 [Asia/Shanghai] [ProxySystem] - [Browser Page Error] [JavaScript Error: "Camoufox can’t establish a connection to the server at ws://127.0.0.1:9998/." {file: "blob:https://4bub4uor6w4pdu09lmbekx7zvne7hx07ezdyv2uci2kvvaixq7-h868144788.scf.usercontent.goog/afaab152-c14f-4acb-bb1d-2961e8bb5264" line: 287}]

[INFO] 2026-02-12 18:49:03.876 [Asia/Shanghai] [ProxySystem] - [Browser]  WebSocket connection error: JSHandle@object

[INFO] 2026-02-12 18:49:03.877 [Asia/Shanghai] [ProxySystem] - [Browser] ❌ Connection disconnected, preparing to reconnect...

[INFO] 2026-02-12 18:49:09.062 [Asia/Shanghai] [ProxySystem] - [Browser] Attempting reconnection 3 attempt...

[INFO] 2026-02-12 18:49:09.063 [Asia/Shanghai] [ProxySystem] - [Browser] Connecting to server: ws://127.0.0.1:9998

[INFO] 2026-02-12 18:50:12.445 [Asia/Shanghai] [ProxySystem] - [Browser]  WebSocket connection error: JSHandle@object

[ERROR] 2026-02-12 18:50:12.444 [Asia/Shanghai] [ProxySystem] - [Browser Page Error] [JavaScript Error: "Camoufox can’t establish a connection to the server at ws://127.0.0.1:9998/." {file: "blob:https://4bub4uor6w4pdu09lmbekx7zvne7hx07ezdyv2uci2kvvaixq7-h868144788.scf.usercontent.goog/afaab152-c14f-4acb-bb1d-2961e8bb5264" line: 287}]

[INFO] 2026-02-12 18:50:12.445 [Asia/Shanghai] [ProxySystem] - [Browser] ❌ Connection disconnected, preparing to reconnect...

[INFO] 2026-02-12 18:50:17.448 [Asia/Shanghai] [ProxySystem] - [Browser] Attempting reconnection 4 attempt...

[INFO] 2026-02-12 18:50:17.449 [Asia/Shanghai] [ProxySystem] - [Browser] Connecting to server: ws://127.0.0.1:9998
```

[INFO] 2026-02-12 19:55:45.975 [Asia/Shanghai] [ProxySystem] - ==================================================

[INFO] 2026-02-12 19:55:45.975 [Asia/Shanghai] [ProxySystem] - ✅ [Browser] Account 0 context initialized successfully!

[INFO] 2026-02-12 19:55:45.975 [Asia/Shanghai] [ProxySystem] - ✅ [Browser] Browser client is ready.

[INFO] 2026-02-12 19:55:45.975 [Asia/Shanghai] [ProxySystem] - ==================================================

[INFO] 2026-02-12 19:55:45.976 [Asia/Shanghai] [ProxySystem] - [System] ✅ Successfully started with account #0!

[INFO] 2026-02-12 19:55:46.071 [Asia/Shanghai] [ProxySystem] - [Browser] System initializing...

[INFO] 2026-02-12 19:55:46.072 [Asia/Shanghai] [ProxySystem] - [Browser] Connecting to server: ws://127.0.0.1:9998

[INFO] 2026-02-12 19:55:47.121 [Asia/Shanghai] [ProxySystem] - [Browser] 🛡️ Background Wakeup Service (Rocket Handler) started...

[ERROR] 2026-02-12 19:55:49.577 [Asia/Shanghai] [ProxySystem] - [Browser Page Error] [JavaScript Error: "Camoufox can’t establish a connection to the server at ws://127.0.0.1:9998/." {file: "blob:https://4bub4uor6w4pdu09lmbekx7zvne7hx07ezdyv2uci2kvvaixq7-h868144788.scf.usercontent.goog/01493ace-5315-4a9a-9bec-26ba6675cf9d" line: 287}]

[INFO] 2026-02-12 19:55:49.577 [Asia/Shanghai] [ProxySystem] - [Browser]  WebSocket connection error: JSHandle@object

[INFO] 2026-02-12 19:55:49.578 [Asia/Shanghai] [ProxySystem] - [Browser] System initialization failed: undefined

[ERROR] 2026-02-12 19:55:49.579 [Asia/Shanghai] [ProxySystem] - [Browser Page Error] Proxy system startup failed: JSHandle@object

[INFO] 2026-02-12 19:55:49.580 [Asia/Shanghai] [ProxySystem] - [Browser] Proxy system startup failed: undefined

[INFO] 2026-02-12 19:55:49.580 [Asia/Shanghai] [ProxySystem] - [Browser] ❌ Connection disconnected, preparing to reconnect...

[INFO] 2026-02-12 19:55:54.587 [Asia/Shanghai] [ProxySystem] - [Browser] Attempting reconnection 1 attempt...

[INFO] 2026-02-12 19:55:54.588 [Asia/Shanghai] [ProxySystem] - [Browser] Connecting to server: ws://127.0.0.1:9998

[ERROR] 2026-02-12 19:55:58.203 [Asia/Shanghai] [ProxySystem] - [Browser Page Error] [JavaScript Error: "Camoufox can’t establish a connection to the server at ws://127.0.0.1:9998/." {file: "blob:https://4bub4uor6w4pdu09lmbekx7zvne7hx07ezdyv2uci2kvvaixq7-h868144788.scf.usercontent.goog/01493ace-5315-4a9a-9bec-26ba6675cf9d" line: 287}]

[INFO] 2026-02-12 19:55:58.204 [Asia/Shanghai] [ProxySystem] - [Browser]  WebSocket connection error: JSHandle@object

[INFO] 2026-02-12 19:55:58.204 [Asia/Shanghai] [ProxySystem] - [Browser] ❌ Connection disconnected, preparing to reconnect...

[INFO] 2026-02-12 19:56:06.695 [Asia/Shanghai] [ProxySystem] - [Browser] Attempting reconnection 2 attempt...

[INFO] 2026-02-12 19:56:06.696 [Asia/Shanghai] [ProxySystem] - [Browser] Connecting to server: ws://127.0.0.1:9998

[ERROR] 2026-02-12 19:56:09.861 [Asia/Shanghai] [ProxySystem] - [Browser Page Error] [JavaScript Error: "Camoufox can’t establish a connection to the server at ws://127.0.0.1:9998/." {file: "blob:https://4bub4uor6w4pdu09lmbekx7zvne7hx07ezdyv2uci2kvvaixq7-h868144788.scf.usercontent.goog/01493ace-5315-4a9a-9bec-26ba6675cf9d" line: 287}]

[INFO] 2026-02-12 19:56:09.861 [Asia/Shanghai] [ProxySystem] - [Browser]  WebSocket connection error: JSHandle@object

[INFO] 2026-02-12 19:56:09.862 [Asia/Shanghai] [ProxySystem] - [Browser] ❌ Connection disconnected, preparing to reconnect...

--- a/src/core/ProxyServerSystem.js
+++ b/src/core/ProxyServerSystem.js
@@ -482,19 +482,32 @@ class ProxyServerSystem extends EventEmitter {
 
     async _startWebSocketServer() {
         return new Promise((resolve, reject) => {
+            let isListening = false;
+
             this.wsServer = new WebSocket.Server({
                 host: this.config.host,
                 port: this.config.wsPort,
             });
-            this.wsServer.on("listening", () => {
+
+            this.wsServer.once("listening", () => {
+                isListening = true;
                 this.logger.info(
                     `[System] WebSocket server is listening on ws://${this.config.host}:${this.config.wsPort}`
                 );
                 resolve();
             });
+
             this.wsServer.on("error", err => {
-                this.logger.error(`[System] WebSocket server failed to start: ${err.message}`);
-                reject(err);
+                if (!isListening) {
+                    this.logger.error(
+                        `[System] WebSocket server failed to start: ${err.message}`
+                    );
+                    reject(err);
+                } else {
+                    this.logger.error(
+                        `[System] WebSocket server runtime error: ${err.message}`
+                    );
+                }
             });
             this.wsServer.on("connection", (ws, req) => {
                 this.connectionRegistry.addConnection(ws, {

